src/botPage/view/logger.js

```diff
@@ -20,9 +20,16 @@ const notify = ({ className, message, position = 'left', sound = 'silent' }) =>
 
     $.notify(message.toString(), { position: `bottom ${position}`, className });
     if (sound !== 'silent') {
-        $(`#${sound}`)
+        const playPromise = $(`#${sound}`)
             .get(0)
             .play();
+        if (playPromise !== undefined) {
+            playPromise
+                .then(() => {
+                    sound.pause();
+                })
+                .catch(() => {});
+        }
```

Comment:
we don't need to define playPromise here :))

Reply:
Actuallt becuse .play() is a promise from now on, it might cause some issues if it is not defined like this for chrome version >= 50. The best practice has been defined on google developer website: 
`https://developers.google.com/web/updates/2016/03/play-returns-promise?hl=en`